### PR TITLE
Create ruleset: WolverineWorldwide.com

### PR DESCRIPTION
**Pull request by injust**

### CODE REVIEW

File: src/chrome/content/rules/WolverineWorldwide.com.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<ruleset name="WolverineWorldwide.com">
+	<target host="wolverineworldwide.com" />
+	<target host="www.wolverineworldwide.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/WolverineWorldwide.com.xml]: `img.wolverineworldwide.com`

Comment (injust) [reply src/chrome/content/rules/WolverineWorldwide.com.xml]: Added, thanks!

File: src/chrome/content/rules/WolverineWorldwide.com.xml [hunk 2]
```diff
@@ -1,6 +1,15 @@
...
+		- img.wolverineworld.com, equivalent to wolverine-o.scene7.com
+-->
+
 <ruleset name="WolverineWorldwide.com">
 	<target host="wolverineworldwide.com" />
 	<target host="www.wolverineworldwide.com" />
+	<target host="img.wolverineworldwide.com" />
+
+	<rule from="^http://img\.wolverineworldwide\.com/"
+		to="https://wolverine-o.scene7.com/" />
```
Comment (J0WI) [inline src/chrome/content/rules/WolverineWorldwide.com.xml]: Please add a test url for redirects to other hosts.

Comment (injust) [reply src/chrome/content/rules/WolverineWorldwide.com.xml]: Added, thanks.

**Pull request merged**